src/components/App.tsx

```diff
@@ -66,7 +67,9 @@ class App extends React.Component {
     try {
       error = JSON.parse(error);
     } catch (e) {}
```

Comment:
What we do here if it cannot be parsed?

Reply:
I didn't write this part, but I assume we try to parse first (maybe it can be a stringified json) and if that fails we use the original version.

Actually I made some testing and this part of the code doesn't work as expected. useUnauthorizedApi has a hook inside it (useSnackbar), and hooks cannot be used this way so it will throw, so this part needs some refactor / review in the future.